x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/EvaluatorImplementer.java

```diff
@@ -114,20 +131,30 @@ private MethodSpec ctor() {
         return builder.build();
     }
 
+    @SuppressWarnings("checkstyle:DescendantToken")
```

Comment:
I'm not sure what this is about but I imagine we should fix the checkstyle warning.